OpenBoard.pro

```diff
@@ -432,6 +437,7 @@ linux-g++* {
     QMAKE_CFLAGS += -fopenmp
     QMAKE_CXXFLAGS += -fopenmp
     QMAKE_LFLAGS += -fopenmp
+	INCLUDEPATH += /usr/include/ffmpeg
```

Comment:
probably unintended

Reply:
No, that's intended. It's necessary for CentOS/RHEL and won't harm with other GNU/Linux distributions.

Reply:
I would add a comment explaining just that and indent with spaces then!
be careful, the commit name doesn't explain your change very well

Reply:
OK, I've documented the change.
I've also noticed, that with Gnome3 ist necessary to modify src/frameworks/UBPlatformUtils_linux.cpp in the same way as src/frameworks/UBPlatformUtils_win.cpp